pkg/uefivars/vars.go

```diff
@@ -0,0 +1,314 @@
+// Copyright (C) 2015-2020 the u-root Authors. All Rights Reserved.
```

Comment:
Is there some way to split this into two packages, such that one doesn't have to pull in all the bootvar parsing dependencies if they don't want them?

pkg/uefivars
pkg/uefivars/bootvars

where bootvars depends on uefivars?

Reply:
See what you think now

Reply:
really nice!